kuma/static/js/contribution-handler.js

```diff
@@ -359,9 +362,10 @@
 
         // Send GA Event.
         mdn.analytics.trackEvent({
-            category: 'Contribution submission',
-            action: isPopoverBanner ? 'On Popover' : 'On Page',
-            value: 1
+            category: 'payments',
+            action: 'submission',
+            label: isPopoverBanner ? 'On pop over' : 'On FAQ page',
+            value: selectedAmount
```

Comment:
This should be ``selectedAmount * 100`` for consistency with the other events with values.